CleverGirl/CleverGirl/Objects/AmmoModeAttackEvaluation.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using CustAmmoCategories;
+
+namespace CleverGirl.Objects;
+
+public class AmmoModeAttackEvaluation
+{
+    public Dictionary<Weapon, AmmoModePair> WeaponList;
+    public AIUtil.AttackType AttackType;
+    public float HeatGenerated;
+    public float ExpectedDamage;
+    public float lowestHitChance;
+
+    public int CompareTo(object otherObj)
```

Comment:
Are you dropping the utility comparison entirely in this?

Reply:
Not sure what you mean? The compare to is used to sort the list of attacks later, which I am not sure why it was done as it iterated over the whole list later anyway, but I kept it.

Reply:
I mean the 'utility' like heat-as-damage, knockdown, etc. It's present at https://github.com/BattletechModders/CleverGirl/blob/11d38af111c0ced70f3e094a3993524502d6a945/CleverGirl/CleverGirl/Helper/AIHelper.cs#L252 when calculating in the EV in the main branch. Wasn't sure from my reading how your changes would intersect with that.